draft-terbu-sd-jwt-vc.md

```diff
@@ -1,9 +1,9 @@
 %%%
-title = "SD-JWT-based Verifiable Credentials with JSON payloads (SD-JWT VC)"
+title = "Verifiable Credentials with a JWT Claim Set with or without Selective Disclosure"
```

Comment:
yeah, title.. `JWT based` and `with a JWT Claim set` is different in my mind because `JWT != SD-JWT`, and the most important thing we are trying to convey is that the payload is vanilla JSON and I found `with a JWT Claim set` to be a good compromise because JSON-LD payload does not use JWT claims while JSON-LD is still JSON.

Reply:
I'd argue that a shorter more succinct title like `JWT and SD-JWT Verifiable Credentials` would be better. Any subtleties could be added/addressed in the abstract or intro. If needed. But the title doesn't seem like the right place to be differentiating a JWT claim set (which is JSON) or JSON or JSON-LD or whatever.

Reply:
And yes, I know there's a recent [title change committed to SD-JWT](https://github.com/oauth-wg/oauth-selective-disclosure-jwt/commit/332ded8faf20d528c5c05e04cebed28aab1a0d33) but I don't love it.

Reply:
> I'd argue that a shorter more succinct title like JWT and SD-JWT Verifiable Credentials would be better. Any subtleties could be added/addressed in the abstract or intro. If needed. But the title doesn't seem like the right place to be differentiating a JWT claim set (which is JSON) or JSON or JSON-LD or whatever.

I'm in support of this.

Reply:
```suggestion
title = "JWT and SD-JWT Verifiable Credentials"
```